src/de/variant_de.rs

```diff
@@ -138,8 +136,8 @@ impl<'de> Deserialize<'de> for Variant {
             }
 
             fn visit_map<V>(self, mut visitor: V) -> Result<Self::Value, V::Error>
-                where
```

Comment:
what's with the formatting all the times 😞

Reply:
I'll make it a hook after we are done with this PR